bin/flutterflow_cli.dart

```diff
@@ -101,6 +111,7 @@ Future _exportCode({
   required String projectId,
   required String destinationPath,
   required bool includeAssets,
+  required bool unzipToParentFolder,
```

Comment:
(Not related to this line, but I need to add this comment somewhere) It interferes with the `--fix` option which assumes the old folder structure. See `final directory = path_util.split(firstFilePath).first;` in `_runFix()`.

Reply:
Thanks for the reminder! Fixed and tested.